easybuild/easyconfigs/g/Gaussian/Gaussian-16.C.02-AVX2.eb

```diff
@@ -0,0 +1,65 @@
+easyblock = 'Tarball'
+
+# IMPORTANT!!
+# Needs to be installed with --umask=007
+
+name = 'Gaussian'
+version = '16.C.02'
+versionsuffix = '-AVX2'
+local_developmentid = 'E6B-432N'
+
+homepage = 'https://www.gaussian.com/'
+
+# For the sake of users I advise changing the last line in the description
+# and versionsuffix to match the actual installed version.
+description = """
+Gaussian provides state-of-the-art capabilities for electronic structure
+modeling. Gaussian 16 is licensed for a wide variety of computer
+systems. All versions of Gaussian 16 contain every scientific/modeling
+feature, and none imposes any artificial limitations on calculations
+other than your computing resources and patience.
+
+This is the official gaussian AVX2 build.
+"""
+
+toolchain = SYSTEM
+
+# Gaussian is licensed software.
+# You need to pick the tar file of your choice from the DVD,
+# put it in your sources tree with a proper name.
+# (should be a tree that ordinary users can't read due to license restrictions)
+# and change the name in "sources"
```

Comment:
we should use `download_instructions` for this, so that a nicer error is given when the sources are not found

Reply:
Also, when I was installing G16 just now, Gaussian had a problem that when the g16 folder (which contains all the Gaussian files) was placed in the software folder, it had 777 permissions. This needed to be changed to 700 before Gaussian was happy to run. Is there a way in the eb file to get easybuild to change the permissions on ```.../easybuild/software/Gaussian/16.C.02-AVX2``` to prevent this issue arising when you first use Gaussian?

Reply:
Also, thats cool I didn't realise there was a ```download_instructions``` entry. As I have just experienced all the things you have to do to install Gaussian recently, I am happy to write something for this.

Reply:
I have just commited an update that includes a ```download_instructions```. Its a bit messy, so make changes as needed.